### ISSUE TITLE
Allow for user-configurable physical constants

### ISSUE THREAD

**Issue by spencerkclark** [spencerahill/aospy#225]
For planetary atmospheres other than Earth's, physical constants like the gravitational acceleration and the planetary radius will be different.  To accommodate results from simulations of these atmospheres, `aospy` will need a mechanism to allow users to set these constants themselves (rather than have them be hard-coded to Earth's values).

xref: @spencerahill's https://github.com/spencerahill/aospy/pull/223#issuecomment-340331253

**Comment by spencerahill** [comment]
This is a particular example of the broader issue of supporting user-specified config, c.f. #145.  But I think it's worth having this separate issue to track this, as #145 is pretty open ended.

**Comment by spencerkclark** [comment]
Right, I see #145 as more global configuration-type stuff; this is likely to vary from `Run` to `Run` or `Model` to `Model`, so it might require a different approach.
